parts/linux/cloud-init/artifacts/init-aks-custom-cloud-mariner.sh

```diff
@@ -16,7 +16,16 @@ cp /root/AzureCACertificates/*.crt /etc/pki/ca-trust/source/anchors/
 
 cloud-init status --wait
 
-# TODO - Set the repoDepotEndpoint in a .repo file if package update becomes necessary
+marinerRepoDepotEndpoint="$(echo "${REPO_DEPOT_ENDPOINT}" | sed 's/\/ubuntu//')"
+if [[ "$marinerRepoDepotEndpoint" == "" ]]; then
```

Comment:
wasn't sure if the fact that its empty is important enough to exit from the entire script. So far its just a warning that gets printed to logs.